configuration/micro_kernel_trait.rst

```diff
@@ -146,31 +148,29 @@ hold the kernel. Now it looks like this::
                 new \Symfony\Bundle\TwigBundle\TwigBundle(),
             ];
 
-            if ($this->getEnvironment() == 'dev') {
+            if ('dev' === $this->getEnvironment()) {
                 $bundles[] = new \Symfony\Bundle\WebProfilerBundle\WebProfilerBundle();
             }
 
             return $bundles;
         }
 
+        protected function build(ContainerBuilder $container)
+        {
+            $container->registerExtension(new AppExtension());
+        }
+
         protected function configureContainer(ContainerConfigurator $c): void
         {
             $c->import(__DIR__.'/../config/framework.yaml');
+            $c->import(__DIR__.'/../config/web_profiler.yaml');
 
             // register all classes in /src/ as service
             $c->services()
                 ->load('App\\', __DIR__.'/*')
                 ->autowire()
                 ->autoconfigure()
             ;
-
-            // configure WebProfilerBundle only if the bundle is enabled
-            if (isset($this->bundles['WebProfilerBundle'])) {
-                $c->extension('web_profiler', [
-                    'toolbar' => true,
-                    'intercept_redirects' => false,
-                ]);
-            }
```

Comment:
Changes in this file do not seem related, 🤔

Reply:
You're right, it's not related and I'm going to revert it to not add noise.